File: iteration2/src/Session.java
import java.time.LocalDateTime;

public class Session {
    // Attributes
    private Person user; // user of the session
    private LocalDateTime startingTime; // start time of the session

    // Constructors
    public Session(Person user) {
        this.user = user;
        this.startingTime = LocalDateTime.now();
    }

    // Methods
    public LocalDateTime getStartingTime() {
        return startingTime;
    }

    public Person getUser() {
        return user;
    }

    public void setUser(Person user) {
        this.user = user;
    }  
    
    public void setStartingTime(LocalDateTime startingTime) {
        this.startingTime = startingTime;
    }
}

File: iteration3/src/session.py
from person import Person

from datetime import datetime

class Session:
    def __init__(self, user: Person):
        self.__user = user
        self.__starting_time = datetime.now()

    def get_starting_time(self):
        return self.__starting_time

    def get_user(self):
        return self.__user

    def set_user(self, user: Person):
        self.__user = user
